specification/securityinsights/SecurityInsights.Management/models.tsp

```diff
@@ -0,0 +1,9171 @@
+   */
+  installedVersion?: string;
+
+  /**
+   * The metadata resource id.
+   */
+  metadataResourceId?: Azure.Core.armResourceIdentifier;
+
+  /**
+   * The json of the ARM template to deploy. Expandable.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-unknown" "For backward compatibility"
+  packagedContent?: unknown;
+}
+
+/**
+ * List of all the template.
+ */
+model ProductTemplateList is Azure.Core.Page<ProductTemplateModel>;
+
+/**
+ * Template property bag.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "For backward compatibility"
+model ProductTemplateProperties {
+  ...TemplateBaseProperties;
+  ...ProductTemplateAdditionalProperties;
+}
+
+/**
+ * Template property bag.
+ */
+model TemplateBaseProperties {
+  /**
+   * Static ID for the content.  Used to identify dependencies and content from solutions or community.  Hard-coded/static for out of the box content and solutions. Dynamic for user-created.  This is the resource name
+   */
+  contentId?: string;
+
+  /**
+   * Unique ID for the content. It should be generated based on the contentId of the package, contentId of the template, contentKind of the template and the contentVersion of the template
+   */
+  contentProductId?: string;
+
+  /**
+   * Version of the package.  Default and recommended format is numeric (e.g. 1, 1.0, 1.0.0, 1.0.0.0), following ARM metadata best practices.  Can also be any string, but then we cannot guarantee any version checks
+   */
+  packageVersion?: string;
+
+  /**
+   * Version of the content.  Default and recommended format is numeric (e.g. 1, 1.0, 1.0.0, 1.0.0.0), following ARM metadata best practices.  Can also be any string, but then we cannot guarantee any version checks
+   */
+  version?: string;
+
+  /**
+   * The display name of the template
+   */
+  displayName?: string;
+
+  /**
+   * The kind of content the template is for.
+   */
+  contentKind?: Kind;
+
+  /**
+   * Source of the content.  This is where/how it was created.
+   */
+  source?: MetadataSource;
+
+  /**
+   * The creator of the content item.
+   */
+  author?: MetadataAuthor;
+
+  /**
+   * Support information for the template - type, name, contact information
+   */
+  support?: MetadataSupport;
+
+  /**
+   * Dependencies for the content item, what other content items it requires to work.  Can describe more complex dependencies using a recursive/nested structure. For a single dependency an id/kind/version can be supplied or operator/criteria for complex formats.
+   */
+  dependencies?: MetadataDependencies;
+
+  /**
+   * Categories for the item
+   */
+  categories?: MetadataCategories;
+
+  /**
+   * Providers for the content item
+   */
+  providers?: string[];
+
+  /**
+   * first publish date content item
+   */
+  firstPublishDate?: plainDate;
+
+  /**
+   * last publish date for the content item
+   */
+  lastPublishDate?: plainDate;
```

Comment:
`lastPublishDate` should generate date rather than string

breaking change 
<img width="812" height="273" alt="image" src="https://github.com/user-attachments/assets/6f3b148d-f330-4b2b-ba3f-7ed0d951471e" />